HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -193,7 +193,8 @@ def self.apply_total_loads_ems_program(model, hpxml_osm_map, hpxml_header)
       duct_zone_names = unit_model.getThermalZones.select { |z| z.isPlenum }.map { |z| z.name.to_s }
       dehumidifier = unit_model.getZoneHVACDehumidifierDXs
       dehumidifier_name = dehumidifier[0].name.to_s unless dehumidifier.empty?
-      defrost_load_oes = unit_model.getOtherEquipments.select { |o| o.name.to_s.include? 'defrost heat load' }
+      # Spaces in other equipment names are replaced with '_' after merging models, include both names
+      defrost_load_oes = unit_model.getOtherEquipments.select { |o| (o.name.to_s.include? 'defrost heat load') || (o.name.to_s.include? 'defrost_heat_load') }
```

Comment:
@yzhou601 Using object names is not a robust way of determining object types (as evidence by the fact that you had to make a fix to the logic). Instead, you should set 'ObjectType' on the `additionalProperties` of the equipment object and check against that. We have lots of examples in the code of doing this, let me know if you want me to point you to one.

Reply:
Much cleaner, thanks!